pkgs/fc/ceph/fc/ceph/util.py

```diff
@@ -0,0 +1,102 @@
+import json
+import subprocess
+from subprocess import PIPE
+
+LVM_QUERY_OPTIONS = ("--reportformat", "json", "--units", "b", "--nosuffix")
+
+
+class JSONRunner(object):
+    def __init__(self, runner):
+        self.runner = runner
+
+    def __run__(self, name, *args, **kw):
+        result = getattr(self.runner, name)(*args, **kw)
+        result = json.loads(result)
+        return result
+
```

Comment:
For an easier superficial code structure, I suggest marking the tool-specific overrides with a comment to make the role of these member functions obvious at first glance. 

```suggestion

# tool-specific overrides of JSON formatting and extraction

```

Reply:
thanks. added more docs as suggested.